PokemonSearch/PokemonSearch/SearchTableViewController.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  SearchTableViewController.swift
+//  PokemonSearch
+//
+//  Created by Jocelyn Stuart on 1/25/19.
+//  Copyright © 2019 JS. All rights reserved.
+//
+
+import UIKit
+
+class SearchTableViewController: UITableViewController, UISearchBarDelegate {
+
+    var pokemonController = PokemonController()
```

Comment:
You don't wanna create 2 different controllers. You already initialized it in `PokemonTableViewController` and pass it through the segue
```suggestion
    var pokemonController: PokemonController?
```